addons/hw_drivers/drivers/KeyboardUSBDriver.py

```diff
@@ -119,13 +119,18 @@ def load_layouts_list(cls):
                 })
 
     def _set_name(self):
+        def _sanitize(value):
```

Comment:
you can do as @qle-odoo said, and remove the useless function "_sanitize" by just calling the replace. (a nested function is not very usefull as we cannot override it, and no need of a function when we only call a function in it)